.github/workflows/release.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches: ['main']
     tags: ['*']
     paths:
+      - '.github/**'
```

Comment:
Why would we want to create a new docker image version whenever `.github` files are changed?

Reply:
Oops, that's a leftover line from my testing. I'll remove.